doc/server/ziohttp.md

```diff
@@ -1,28 +1,52 @@
-# Running as an zio-http server
+# Running as a zio-http server
 
-To expose an endpoint as a [zio-http](https://github.com/dream11/zio-http) server, first add the following dependency:
+The `tapir-zio` module defines type aliases and extension methods which make it more ergonomic to work with
+[ZIO](https://zio.dev) and tapir. Moreover, `tapir-zio-http-server` contains an interpreter useful when
+exposing the endpoints using the [ZIO Http](https://github.com/dream11/zio-http) server.
+
+You'll need the following dependency for the `ZServerEndpoint` type alias and helper classes:
+
+```scala
+"com.softwaremill.sttp.tapir" %% "tapir-zio" % "@VERSION@"
+```
+
+or just add the zio-http integration which already depends on `tapir-zio`:
 
 ```scala
 "com.softwaremill.sttp.tapir" %% "tapir-zio-http" % "@VERSION@"
 ```
 
-Now import the object:
+Next, instead of the usual `import sttp.tapir._`, you should import (or extend the `ZTapir` trait, see [MyTapir](../mytapir.md)):
+
+```scala mdoc:compile-only
+import sttp.tapir.ztapir._
+```
+
+This brings into scope all of the [basic](../endpoint/basics.md) input/output descriptions, which can be used to define an endpoint.
+Additionally, it defines the `ZEndpoint` type alias, which should be used instead of `Endpoint`.
```

Comment:
This line should probably be removed as there is no `ZEndpoint` anymore?

Reply:
good point, done :)